pkg/partition/partprobe/builder.go

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
n_p: moving this to package pkg/pool/partition can help organize a little better. 
What are your thoughts?

Reply:
Also, file name builder.go becomes too generic IMO.

Reply:
`partprobe.go` sounds good to me and the `predicate.go` content can move here. 
Idea is to avoid getting into imparting the creational patterns.

Reply:
Sounds good